### ISSUE TITLE
Option to remove instruments (0xtob/#33)

### ISSUE THREAD

**Issue by asiekierka** [asiekierka/nitrotracker#54]
>Doesn't seem too hard to add as a feature. This would save RAM by removing unnecessary waveforms. The only way to remove an instrument currently is to either overwrite it with another one, or to zap all of them.

**Comment by deadpahn** [comment]
YES. Delete unused instruments would be helpful. Although I have to say I'm
not running into the same problems with the unofficial build of NT. OG NT
.... I really had that RAM bar filled some days.

**Comment by asiekierka** [comment]
There have been many memory leak fixes, as well as shaving off a few hundred KB in the tracker's own RAM usage, as well as adding DSi mode support (which technically allows up to 16MB of RAM - especially useful with the undo/redo buffer).
